src/HaskellCI/Config.hs

```diff
@@ -185,6 +187,8 @@ configGrammar = Config
         ^^^ metahelp "JOB" "Jobs to additionally build with OSX"
     <*> C.monoidalFieldAla    "apt"                       (alaSet' C.NoCommaFSep C.Token')    #cfgApt
         ^^^ metahelp "PKG" "Additional apt packages to install"
+    <*> C.monoidalFieldAla    "patches"                   (C.alaList' C.FSep C.Token')        #cfgPatches
```

Comment:
I'd use `NoCommaFSep` here too. Allowing commas is redundant.

Reply:
And let's name the field `travis-patches`

Reply:
Done and done.